jekyll/_cci2/set-environment-variable.adoc

```diff
@@ -35,7 +35,7 @@ jobs:
           # available in current step.
           command: |
             echo 'export PATH=/path/to/foo/bin:"$PATH"' >> "$BASH_ENV"
-            echo 'export VERY_IMPORTANT=$(cat important_value)' >> "$BASH_ENV"
+            echo "export VERY_IMPORTANT=$(cat important_value)" >> "$BASH_ENV"
```

Comment:
Hi @legopin , so I've gotten feedback from our engineers and their advice is to actually leave out the bash command substitution (`$(...)`) out from examples entirely. So perhaps the better edit to do here would be something like what you indicated in the PR description e.g.
```
"export VERY_IMPORTANT=VALUE_CONTENT"
```
Does that make sense?